test/index.ts

```diff
@@ -0,0 +1,173 @@
+import { Calculator, Operation } from '../src/Calculator';
```

Comment:
no space before and after {}
`import {Calculator, Operation} from '../src/Calculator`